strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NugetArtifactGenerator.java

```diff
@@ -33,107 +32,73 @@
  * @author Kate Novik.
  * @author Pablo Tirado
  */
-public class NugetArtifactGenerator
-        implements ArtifactGenerator
-{
+public class NugetArtifactGenerator implements ArtifactGenerator {
```

Comment:
Please, preserve the original formatting.

Reply:
My IDEA is using a different format even after setting the eclipse from the doc, I'm fixing this